content/src/main/content/jcr_root/apps/acs-tools/components/dumplibs/clientlibs/js/components/clientlib-modal.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * #%L
+ * ACS AEM Tools Package
+ * %%
+ * Copyright (C) 2014 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+/*global Vue, axios, console */
+
+(function () {
+    'use strict';
+    Vue.component('clientlib-modal', {
+        data: function () {
+            return {
+                open: false,
+                clientlib: {},
+                clientlibs: {
+                    js: [],
+                    css: []
+                },
+                header: "",
+                loading: true,
+                error: false
+            };
+        },
+        methods: {
+            getClientlibs: function (params) {
+                return axios.get('/bin/acs-tools/dumplibs.json', {
```

Comment:
This should change to `/etc/acs-tools/dumplibs.xxx.json` ... ideally, it would support custom servlet contexts, but not sure if there's a simple/clean way to expose that to the Vue app. If not, we should just make sure to document it doesn't support custom contexts..

Reply:
I dont know what you mean by `custom servlet contexts` ?

Reply:
If you run aem on http://host:port/custom-context/etc/acs-tools/dumplibs.html

Reply:
@davidjgonzalez still not getting what you mean? is this a sling/java servlet concept that I've never heard of? or is it some kind of pattern? Also, are there an examples I can reference?